src/pages/claim-review.tsx

```diff
@@ -55,7 +55,7 @@ const ClaimPage: NextPage<{
                 jobTitle: personality.description,
                 image: personality.image,
             },
-            datePublished: claim.date,
+            datePublished: sentence.date,
```

Comment:
Are you sure the sentence date is the same as the claim review date? Also, are they the same as the publish date in the state machine?

Reply:
The sentence date has the claim review date
```js
return Promise.all([
    this.claimReviewService.getReviewStatsBySentenceHash({
        sentence_hash,
        isDeleted: false,
        isPublished: true,
    }),
    this.claimReviewService.getReviewBySentenceHash(sentence_hash),
    this.claimReviewService.getUserReviewBySentenceHash(
        sentence_hash
    ),
]).then(([stats, claimReviewObj, userReview]) => {
    return {
        userReview,
        date: claimReviewObj.date,
        personality: claimReviewObj.personality,
        stats,
        ...sentence.toObject(),
    };
});
```
## Claim Review Published date
![image](https://user-images.githubusercontent.com/73478823/184015333-54f9e4a0-a016-48df-91da-99adcc900826.png)

## claim Review Task published date
![image](https://user-images.githubusercontent.com/73478823/184015149-888ea0d9-6fc4-45fb-9d8c-08077fc60519.png)

Reply:
miliseconds diference